src/library/impl/meta_reader/view.h

```diff
@@ -237,7 +242,7 @@ namespace xlang::meta::reader
                 return{};
             }
 
-            handle mapping{ CreateFileMappingA(file.value, nullptr, PAGE_READONLY, 0, 0, nullptr) };
+            handle mapping{ CreateFileMapping(file.value, nullptr, PAGE_READONLY, 0, 0, nullptr) };
```

Comment:
Any reason for this change? This makes the code dependent on whether or not UNICODE is defined. Avoid using these function-like macros and explicitly pick the function you wish to call.

Reply:
I was trying to use the meta data reader in a c++/winrt app and hitting the limitation that the A version isn't available.  Would you recommend doing something like this...
#ifndef UNICODE
    // use CreateFileMappingA
#else 
    // use CreateFileMappingW
#endif

Reply:
You always want CreateFileMappingW with this change - just make that explicit

---
In reply to: [306941300](https://github.com/microsoft/xlang/pull/536#discussion_r306941300) [](ancestors = 306941300)

Reply:
Yep what Scott said.